src/pages/Index/index.tsx

```diff
@@ -1,27 +1,44 @@
 import * as React from 'react';
-import { Link } from 'react-router-dom';
 import { Col } from 'react-bootstrap';
 
-type PublicProps = {};
+import { gettext } from '../../utils';
+
+type Props = {
+  apiHost: string;
+  repoUrl: string;
+};
+
+export class IndexBase extends React.Component<Props> {
+  static defaultProps = {
+    apiHost: process.env.REACT_APP_API_HOST,
+    repoUrl: 'https://github.com/mozilla/addons-code-manager',
+  };
 
-export class IndexBase extends React.Component<PublicProps> {
   render() {
+    const { apiHost, repoUrl } = this.props;
+
     return (
       <Col>
         <p>
-          There is nothing you can do here, but try{' '}
-          <Link to="/en-US/browse/494431/versions/1532144/">
```

Comment:
It'd be nice to use the reviewers website instead because we're relying a lot on these two examples and we can easily miss issues because of that. It was useful before we could get a valid link to a version, but the reviewers tool now has a link to the code-manager, so we could just use that instead. That would add some more entropy into the test dataset.

Reply:
I agree but it's hard to find a decent example of an add-on (in dev especially) that shows many folders and many different types of notices (global, inline). I think if we had these links at the bottom it would be a good escape hatch for when it's hard to find complete add-ons.